bundle/cert-manager-operator/manifests/cert-manager-config.crd.yaml

```diff
@@ -1,51 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
something's wrong, this file should not have been removed (no matter if it's replaced by something else or not)

Reply:
I'm not entirely sure how @s-urbaniak generated those files but I think he used some old version of the Controller Gen (or Operator SDK). Over the time the naming convention changed.